### ISSUE TITLE
Unable to run WinAppDriver with https://github.com/microsoft/WinUI-3-Demos

### ISSUE THREAD

**Issue by ArnOmsft** [microsoft/microsoft-ui-xaml#3002]
This is an issue reported by a customer.
The customer tried to run WinAppDriver on our WinUI 3 sample (https://github.com/microsoft/WinUI-3-Demos) on Intel platform, and WinAppdriver is not able to identify and launch the application.

**Comment by llongley** [comment]
I believe this is a WinAppDriver issue.  I've ported this issue to their repo here:

https://github.com/microsoft/WinAppDriver/issues/1261
